### PR TITLE
fix padding mode in artcnn

### PR DESCRIPTION
**Pull request by umzi2**
I somehow made a mistake a long time ago, now the official [implementation](https://github.com/Artoriuz/ArtCNN/blob/main/Notebooks/ArtCNN_R_PyTorch.ipynb) has come out, and I noticed it. I’m actually not sure that they even launched it, in terms of 

```
self.feats_conv = nn.Conv2d(filters, 4, kernel_size=kernel_size, padding='same')
self.pixel_shuffle = nn.PixelShuffle(upscale_factor)
```
it doesn't work that way

### CODE REVIEW

**Comment by the-database** [comment]
Thanks for the PR, looks like it has errors due to passing the padding param to Conv2d twice.

**Comment by umzi2** [comment]
> Thanks for the PR, looks like it has errors due to passing the padding param to Conv2d twice.

my mistake, fixed

**Review by the-database — APPROVED** [review]
Thanks!
